packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.7.0-nullsafety
+* Migrate to nullsafety
+* Breaking Changes:
+    * Removed the deprecated methods: `ImagePicker.pickImage`, `ImagePicker.pickVideo`,
+`ImagePicker.retrieveLostData`
+
+* iOS: update XCUITests to separate each test session.
```

Comment:
This looks like it was accidentally duplicated from the previous notes.

Reply:
How do you suggest I put it?

Reply:
I'm not sure what you're asking. The solution to having accidentally copied and pasted a line from the previous version's release notes is to remove the accidental duplicate.

Reply:
Ahh!, I'm sorry I missed that line. I thought you were talking about the way I put the breaking changes and thought some wording needed to be changed. I'll fix that.